solr/core/src/test/org/apache/solr/handler/admin/ShowFileRequestHandlerTest.java

```diff
@@ -84,7 +84,7 @@ public void testDirList() throws SolrServerException, IOException {
   public void testGetRawFile() throws SolrServerException, IOException {
     SolrClient client = getSolrClient();
     //assertQ(req("qt", "/admin/file")); TODO file bug that SolrJettyTestBase extends SolrTestCaseJ4
-    QueryRequest request = new QueryRequest(params("file", "managed-schema.xml"));
+    QueryRequest request = new QueryRequest(params("file", "managed-schema"));
```

Comment:
Changed your mind?

Reply:
Yeah, for now, till I write a better integration style test.   I discovered that while the unit tests might all pass perfectly, when I fired up solr in cloud mode and non cloud mode it wasn't working ;-(    So I went back to the old way to make srue I was properly testing.